src/request/JsonRpcRequestAggregate.php

```diff
@@ -6,7 +6,7 @@
 
 class JsonRpcRequestAggregate
 {
-    private $requests;
+    private $requests = [];
```

Comment:
I think that initialising fields in the declaration part isn't the prettiest way to do it :)

Reply:
It is for explicity, because it is declared to be always array. it is protection from incorrect state